notebooks/delta_table_upload_demo.ipynb

```diff
@@ -0,0 +1,492 @@
+      "metadata": {
+        "id": "gVFX2uY69u4n"
+      },
+      "execution_count": 3,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The spark_config argument is only required when connecting to a third-party storage service such as AWS S3 or GCS, the fields that need to be in spark_config are:\n",
+        "\n",
+        "\n",
+        "*   jars - A string containing the path to the .jar files needed for the connector you are using, separated by a comma. The .jar file versions will need to be compatible with the version of Spark you have installed.\n",
+        "\n",
+        "  *   AWS S3 - hadoop-aws-2.7.4.jar, aws-java-sdk-1.7.4.jar\n",
+        "  *   GCS - gcs-connector-hadoop3-latest.jar\n",
+        "*   credentials - Only required if connecting to GCS, the value should be the path to your [GCS service account](https://developers.google.com/workspace/guides/create-credentials#create_credentials_for_a_service_account) credentials JSON file. To run this notebook with a GCS credentials file, click on the folder icon on the left sidebar and then click on the upload file icon. This will upload the file to the Notebook's `/content/` folder.\n",
+        "\n",
+        "\n",
+        "*   AWS_ACCCESS_KEY - Only required if connecting to AWS S3, the value should be your [AWS service account](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) access key\n",
+        "*   AWS_SECRET_KEY - Only required if connecting to AWS S3, the value should be your [AWS service account](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) secret key\n",
+        "\n"
+      ],
+      "metadata": {
+        "id": "y8b9Awq7YAfg"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "from pyspark.sql import SparkSession\n",
+        "from delta import configure_spark_with_delta_pip\n",
+        "\n",
+        "SPARK_HOME = '/usr/local/lib/python3.10/dist-packages/pyspark/'\n",
+        "\n",
+        "AWS_ACCESS_KEY = ''\n",
+        "AWS_SECRET_KEY = ''\n",
+        "\n",
+        "aws_spark_config = {\n",
+        "                \"jars\": f\"{SPARK_HOME}/jars/hadoop-aws-2.7.4.jar, {SPARK_HOME}/jars/aws-java-sdk-1.7.4.jar\",\n",
+        "                \"AWS_ACCESS_KEY\": AWS_ACCESS_KEY,\n",
+        "                \"AWS_SECRET_KEY\": AWS_SECRET_KEY\n",
+        "}\n",
+        "\n",
+        "gcs_credentials_file = '/content/{name of credentials .json file}'\n",
+        "\n",
+        "gcs_spark_config = {\n",
+        "                \"jars\": f\"{SPARK_HOME}/jars/gcs-connector-hadoop3-latest.jar\",\n",
+        "                \"credentials\": gcs_credentials_file\n",
+        "}\n",
+        "\n",
+        "#table_path can either be a local file, or a link to a third-party cloud storage service (e.g gs://{bucket_name}/{folder}/{table} or s3a://{bucket_name}/{folder}/{table})\n",
+        "table_path = \"\""
```

Comment:
Include the local path '/content/demo-table' here, if this cell is run after the data table creation step.

Reply:
done